proto/api_test.go

```diff
@@ -46,10 +46,10 @@ func TestResponseHeaderSetGoError(t *testing.T) {
 	rh := ResponseHeader{}
 	rh.SetGoError(&testError{})
 	err := rh.GoError()
-	if reflect.TypeOf(err) != reflect.TypeOf(&GenericError{}) {
-		t.Errorf("expected set error to be type GenericError; got %s", reflect.TypeOf(err))
+	if reflect.TypeOf(err) != reflect.TypeOf(&Error{}) {
```

Comment:
I doubt there's any particular reason; it might have been copy/pasted from another similar test where the expected type is not a compile-time constant. I've changed it.